2024-11-13 16:25:52.985 +03:00 [Error] {"ExceptionMessage":"An error occurred while saving the entity changes. See the inner exception for details.","FullName":null,"MethodName":"Handle","User":"System Admin","Parameters":[{"Name":"request","Value":{"UserId":0,"CitizenId":0,"FullName":"YAVUZ","Email":"string","MobilePhones":"string","Status":true,"BirthDate":"2024-11-13T13:25:24.565Z","Gender":0,"RecordDate":"2024-11-13T13:25:24.565Z","Address":"string","Notes":"string","UpdateContactDate":"2024-11-13T13:25:24.565Z","Password":"string"},"Type":"CreateUserCommand"},{"Name":"cancellationToken","Value":{"IsCancellationRequested":false,"CanBeCanceled":false,"WaitHandle":{"Handle":{"value":3436},"SafeWaitHandle":{"IsInvalid":false,"IsClosed":false}},"Type":"CancellationToken"}]}
